### PR TITLE
AG-9185 Fix histogram remove animation

### PR DESCRIPTION
**Pull request by lsjroberts**
https://ag-grid.atlassian.net/browse/AG-9185

### CODE REVIEW

File: packages/ag-charts-community/src/chart/series/cartesian/histogramSeries.ts [hunk 1]
```diff
@@ -691,14 +691,17 @@ export class HistogramSeries extends CartesianSeries<SeriesNodeDataContext<Histo
...
-            removedIds[d.join('_')] = true;
-        });
+        // Zip an array into an object of keys with a given value
+        const zipObject = (props: Array<any>, value = true) => {
+            const zipped: { [key: string]: boolean } = {};
+            for (let i = 0; i < props.length; i++) {
+                zipped[`${props[i]}`] = value;
+            }
+            return zipped;
+        };
```
Comment (lsjroberts) [inline packages/ag-charts-community/src/chart/series/cartesian/histogramSeries.ts:701]: Where did we discuss would be a good place for this kind of util?

Comment (alantreadway) [reply packages/ag-charts-community/src/chart/series/cartesian/histogramSeries.ts:701]: Perhaps in it's own module in https://github.com/ag-grid/ag-charts/tree/25ba7de8971ec1d70fc635c2cee359b051e9fb87/packages/ag-charts-community/src/util ?

**Comment by lsjroberts** [comment]
Closing as I got the ticket number wrong

**Comment by lsjroberts** [comment]
Redirected to https://github.com/ag-grid/ag-charts/pull/130

**Pull request closed without merging**